app/test_main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import pytest
+
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize("connection_result, url_result, expected_result", [
+    (True, True, "Accessible"),
+    (False, True, "Not accessible"),
+    (True, False, "Not accessible"),
+    (False, False, "Not accessible"),
+])
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(
+        mocked_connection: mock,
+        mocked_url: mock,
```

Comment:
still wrong order of parameters 

@mock.patch("app.main.valid_google_url") -> mocked_url
@mock.patch("app.main.has_internet_connection") -> mocked_connection

should be in the same order as the decorators are applied